models/ios/logging_global/ios_logging_global.yaml

```diff
@@ -0,0 +1,661 @@
+module: ios_logging_global
+version_added: 2.0.1
+short_description: Manages logging attributes of Cisco IOS network devices
+description: This module manages the logging attributes of Cisco IOS network devices.
+author: Sagar Paul (@KB-perByte)
+options:
+  config:
+    description: A list containing dictionary of logging options
+    type: list
+    elements: dict
+    suboptions:
+      hostname: 
+        description: IP address of the logging host
+        type: str
+      buffered:         
+        description: Set buffered logging parameters
+        type: dict
+        suboptions:
+          level:           
+            description: Logging severity level (0-7)
+            type: int
+            choices: [0,1,2,3,4,5,6,7]
+          size:            
+            description: Logging buffer size
+            type: int
+          severity:
+            description: Logging severity level
+            type: str
+            choices: ["alerts","critical","debugging","emergencies","errors","informational","notifications","warnings"]
+          discriminator:    
+            description: Establish MD-Buffer association
+            type: str
+          filtered:         
+            description: Enable filtered logging
+            type: bool
+          xml:              
+            description: Enable logging in XML to XML logging buffer
+            type: bool
+          enable:
+            description: to enable buffered logging
+            type: bool
+      buginf:          
+        description: Enable buginf logging for debugging
+        type: bool
+      cns_events:   
+        description: Set CNS Event logging level
+        type: dict
+        suboptions:
+          level:           
```

Comment:
you can reuse existing level option with & and *. check https://github.com/ansible-network/resource_module_models/blob/master/models/iosxr/bgp_neighbor_address_family/iosxr_bgp_neighbor_address_family.yaml as_override attribute for referanace.

Reply:
@ashwini-mhatre Thanks